.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Report a bug
+about: Report a bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Thank you for reporting this issue! -->
+
+Let's make sure issue is not already fixed in latest builds first.
```

Comment:
From my understanding, the template implicitly assumes that one starts with describing the issue. Maybe the template could start with a "This is the bug" section before the "Steps to reproduce" one?

Reply:
The title of the issue is supposed to brielfly covers the "This is the bug".  The bug description is the final step of the Steps to Reproduce.